projects/listening-comp/services/transcript-processor/app.py

```
@@ -17,6 +25,42 @@
+# Initialize ServiceOrchestrator
+service_orchestrator = ServiceOrchestrator(device=DEVICE)
+
+# Define service dependencies
+question_service = MicroService(
+    name="question_service",
+    host="question-module",  # Docker service name
+    port=5001,
+    endpoint="/api/generate-questions",
+    service_type=ServiceType.PROCESSOR,
+    use_remote_service=True,
+    device=DEVICE
```

Ruff (0.8.2)

Undefined name `DEVICE`

(F821)